addons/mail/models/mail_message.py

```diff
@@ -28,37 +28,37 @@ class Message(models.Model):
       * notification_status
         'ready', 'sent', 'bounce', 'exception', 'canceled'
       * notification_type
-            'inbox', 'email', 'sms' (SMS addon), 'snail' (snailmail addon)
+        'inbox', 'email', 'sms' (SMS addon), 'snail' (snailmail addon)
       * failure_type
-            # mail
-            "SMTP", "RECIPIENT", "BOUNCE", "UNKNOWN"
-            # sms (SMS addon)
-            'sms_number_missing', 'sms_number_format', 'sms_credit',
-            'sms_server', 'sms_acc'
-            # snailmail (snailmail addon)
-            'sn_credit', 'sn_trial', 'sn_price', 'sn_fields',
-            'sn_format', 'sn_error'
+        # mail
+        "SMTP", "RECIPIENT", "BOUNCE", "UNKNOWN", "mail_missing"
```

Comment:
"mail_missing" could be confusing IMO, it's not a mail that is missing but an address.
Why not go for something explicit like "recipient_address_missing" or "mail_address_missing" or something?

Reply:
It was mainly to keep keys short in db. Its label gives an explanation (grepping it returns few occurences in code so easy to find it).